app/controllers/api/base_controller.rb

```diff
@@ -193,6 +193,8 @@ def api_error(type, error)
       # We don't want to return the stack trace, but only log it in case of an internal error
       api_log_error("\n\n#{error.backtrace.join("\n")}") if type == :internal_server_error && !error.backtrace.empty?
 
+      logger.fatal("Error caught: [#{error.class.name}] #{error.message}\n#{error.backtrace.join("\n")}")
```

Comment:
Do we really want to log a stack trace on each error, i.e. a 400 or 403, etc ? I would expect the backtrace only if we have a 500 (:internal_server_error)

Reply:
Right, I would log them all, but only add the \n#{error.backtrace.join("\n")} for internal_server_error.